src/common/vars/static-files-directory.global-override-for-injectable.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getGlobalOverride } from "../test-utils/get-global-override";
+import staticFilesDirectoryInjectable from "./static-files-directory.injectable";
+
+
+export default getGlobalOverride(staticFilesDirectoryInjectable, () => "some-static-directory");
```

Comment:
This needs to be an absolute path

Reply:
It could just be `/some-static-directory`

Reply:
Fixed.